### ISSUE TITLE
Welcome dialog: colour contrast of text

### ISSUE THREAD

**Issue by DavidBailes** [audacity/audacity#8838]
### Bug description

There are problems with the colour contrast of both the slide title and the links: view tutorials, and visit our forum.
The problems with the slide title are only present for the dark and high contrast themes, and these affect all users - the text is either very difficult to see or invisible.
The problem with the colour contrast of the links is more an accessibility issue, and is present in all themes.

### Steps to reproduce

1. Open Audacity having previously set Audacity to use a particular theme.
2. Check the colour contrast of the slide title text, and the text of the links. (eg by using the Accessibility Insights for windows app)

### Expected behavior

For all themes, the colour contrast of the slide title, which is large text, should be at least 3:1.
For all themes, the colour contrast of the link text should be at least 4.5:1

### Actual behavior

Colour contrast of slide title: classic 19.9:1, light 19.8:1, dark 1.5:1, high contrast 1:1 (ie invisible).
Colour contrast of links: classic 3.2:1, light 4.2:1, dark 1.6:1, high contrast 3.3:1.

### Audacity Version

release-3.7.4

### Operating system

Windows

### Additional context

For example, this is using the dark theme:

![Image](https://github.com/user-attachments/assets/4e23968b-2376-4cac-89b1-d9ce3a1c9e17)

See also: https://github.com/audacity/audacity/issues/8836
